e2e-test/trace/build.gradle

```diff
@@ -31,11 +31,10 @@ task removeDataPrepperNetwork(type: DockerRemoveNetwork) {
 }
 
 def RAW_SPAN_PIPELINE_YAML = "raw-span-e2e-pipeline.yml"
-def RAW_SPAN_PIPELINE_EVENT_TYPE_YAML = "raw-span-e2e-pipeline-event-type.yml"
-def RAW_SPAN_PIPELINE_PEER_FORWARDER_EVENT_TYPE_YAML = "raw-span-e2e-pipeline-peer-forwarder-event-type.yml"
+def RAW_SPAN_PIPELINE_PEER_FORWARDER_YAML = "raw-span-e2e-pipeline-peer-forwarder.yml"
+// TODO: modify the content of this file to be compatible with 2.0 in future release
```

Comment:
What do we need to do to keep this compatible with 2.0? Should this be captured as part of the core peer forwarder work?

Reply:
Currently the breaking change in pipeline config is as follows:
1.5 -> 2.0
record_type -> removed
otel_trace_raw_prepper -> otel_trace_raw
otel_trace_group_prepper -> otel_trace_group

>Should this be captured as part of the core peer forwarder work?
Good call out. This compatibility test is only for peer-forwarder plugin, which I am not sure about the timeline of removal (@dlvenable input here?). But with core peer forwarder, we need to device new e2e tests. For 2.0, there will be no backward compatibility test since DataPrepper 1.x does not support core peerforwarder.

Reply:
We are planning to remove the old `peer-forwarder` plugin in 2.0.

Reply:
In that case I will change the TODO comment a bit. The e2e tests will need some major changes once core peer-forwarder is ready.